mobilizon_reshare/main/pull.py

```diff
@@ -0,0 +1,25 @@
+import logging.config
+
+from mobilizon_reshare.event.event import MobilizonEvent
+from mobilizon_reshare.mobilizon.events import get_mobilizon_future_events
+
+from mobilizon_reshare.storage.query.write import (
+    create_unpublished_events,
+)
+
+logger = logging.getLogger(__name__)
+
+
+async def pull() -> list[MobilizonEvent]:
+    """
+    Fetches the latest events from Mobilizon and stores them.
+    :return:
+    """
+
+    # Pull future events from Mobilizon
+    future_events = get_mobilizon_future_events()
+    logger.info(f"Pulled {len(future_events)} from Mobilizon.")
+    # Store in the DB only the ones we didn't know about
+    events = await create_unpublished_events(future_events)
+    logger.info(f"There are now {len(events)} unpublished events.")
```

Comment:
This could be debug level

Reply:
I think some kind of feedback could be useful, both for the user and for tests. What about keeping only the first?

Reply:
the first is fine as INFO. The second one seems a bit excessive.